Updated recipes, add items, fix textures, Added titanium ores, fixed ore gen

Lightning323 opened the pull request:
# Ores

The frequency of titanium ores is currently as follows
moon ---------------- gold rarity
mars ----------------- between gold and iron rarity
mercury ------------- iron rarity
venus ---------------- iron rarity


## Fixed:
Fixed diamond ores not showing up on some planets
Fixed iron ores not showing up on some planets
Fixed misspellings in worldgen tag names

# Items and recipes

Changed recipe for titanium, added some new liquids and items, like chlorine

Lightning323 committed:
Updated recipes

Lightning323 committed:
Update sequenced_titanium.json

Lightning323 committed:
Made recipes easier, fixed space suit textures

Lightning323 committed:
Made recipes easier, fixed space suit textures

zipCoder933 commented:
Fixed it

Lightning323 committed:
Merge branch 'reduxDev' of https://github.com/zipCoder933/Northstar-True-North into reduxDev

Lightning323 committed:
Small tweaks

Lightning323 committed:
Recipe fixes

- Compressed sand bar didnt really work unless there is something else there, added molten sand to combat the issue of recipe conflicts, and to add an extra step to the assembly line

Lightning323 committed:
Fix 2 perhaps?

Lightning323 committed:
Fixed recipes

- Added chlorine, a material that makes producing titanium easier, Chlorine is only make-able after you get to mars (due to electrolysys manufacturing of chlorine)

Lightning323 committed:
updated levels

Lightning323 committed:
Remove final washing step and impure titanium

Lightning323 committed:
Updated recipes

Still havent updated the levels of items yet, but this looks like a very promising one

Lightning323 committed:
updated recipes

Lightning323 committed:
Merge branch '1.20.1/dev' into reduxDev

zipCoder933 commented:
I did my best to ensure it's not too hard and not overly easy. However the only real way to know is after play testing the game.

Lightning323 committed:
updated electrolysys machine texture and model

Lightning323 committed:
updated electrolysys machine texture and model

Lightning323 committed:
Merge branch 'reduxDev' of https://github.com/zipCoder933/Northstar-True-North into reduxDev

Lightning323 committed:
updated models and textures

Lightning323 committed:
added titanium ores, fixed ore mispellings on datapack

Lightning323 committed:
Update shovel.json

Lightning323 committed:
updated models, added liquid carbon, fixed recipes, mars ore, bamboo to carbon

- fixed some models
- added carbon
- changed some recipes
- removed sodium catalyst (It only had a single use, if I find 2 uses it may come back but IDK)
- bamboo + carbon makes bonemeal,
- bamboo can make carbon (maybe this isnt a good idea, due to already being able to obtain carbon in space?)

Lightning323 committed:
fixed models and recipes

- fixed models
- added sodium catalyst back (it has multiple uses now)
- Fixed carbon -> hydrocarbon recipe

Lightning323 committed:
Small tweaks

Lightning323 committed:
fixed temp regulator texture

Lightning323 committed:
Merge remote-tracking branch 'upstream/1.20.1/dev' into redux/recipeChanges

I need to fix landing bug and Mind as well do this
from the same PR as this one

Lightning323 committed:
fixed landings

Landings now disassemble after 20 ticks to prevent clipping through floor
Fixed solar panel texture and model
removed duplicate recipes

Lightning323 committed:
fixed

- Fixed models
- fixed textures

Lightning323 committed:
Merge branch '1.20.1/dev' into reduxDev

Lightning323 committed:
crash bugfix

Lightning323 committed:
Merge branch 'reduxDev' of https://github.com/zipCoder933/Northstar-True-North into redux/recipeChanges

Lightning323 committed:
Update RocketContraptionEntity.java

zipCoder933 commented:
I have fixed the bug where if you bring entities with you into space, they actually make it with you on the way down, 
Before the dimension change caused the unseated entities to not be added on the server

Lightning323 committed:
Fixed bug with unseated entities getting lost on the server after dimension change

zipCoder933 commented:
You can merge it now

Lightning323 committed:
added biofuel texture

- updated models and textures
- small landing fix
- added create desiel generators

Lightning323 committed:
Fixed fuel recipes, renamed an item

- changed charred biomass to dry_plant_fiber
- Since plant fibers primary use is to make bone meal, players should be discouraged form making biodfuel out of it especially considering biofuel requires bone meal (That is assuming biofuel can be made out of dry plant fibers at all)
- added CDG biodesiel as a compatable rocket fuel

Lightning323 committed:
Merge remote-tracking branch 'upstream/1.20.1/dev' into redux/recipeChanges

Lightning323 committed:
Removal of my seating code

- Fixed combustion engine recipe

Lightning323 committed:
updated jet thruster model and texture to a multiblock style
